src/layers/core/grid-layer/grid-layer-vertex.glsl

```diff
@@ -28,6 +28,13 @@ uniform float elevationScale;
 // Result
 varying vec4 vColor;
 
+// whether is point picked
+float isPicked(vec3 pickingColors, vec3 selectedColor) {
```

Comment:
Nit: I might be wrong but think it is possible to compare vectors directly without explicitly doing element wise comparison.

```
return pickingColors == selectedColor;
```